### DISCUSSION TITLE
I need to reset user password for th whole tpot

### DISCUSSION THREAD

**Discussion by Ahmadhashem86** [telekom-security/tpotce#959] [Q&A]
I have installed tpot on Debaian buster using post installation guide!

but It didn't asked me about user credits!
How Can I do now so I donät have user and password! Is there any defaults?

**Comment by rxerium** [comment]
Did you find the solution for this?

**Comment by FloppyDucks** [comment]
Are you asking like how to set up a local account so that you can ssh into the linux box? are you asking like say you want to set up a default local account on your linux box named "TPOT-admin" and use that account to login with, run process with, and manage the box with? 

if so you just need to make a new local account on the server, and log in with that. OR you can log in with the default local account that tpot creates at time of install. 

if you cat /etc/passwd you should see a user tpot:x:2000:2000::/nonexistent:/bin/false (home and shell may differ) but this is the local acct that tpot will auto install. and create, if you run "sudo usermod -s /bin/bash tpot" that will set tpot login shell to /bin/bash. and then 
"sudo usermod -d /home tpot" will assign the /home directory to their home directory. 

Then again I could have completely misunderstood what you wanted, that is very possible, lemme know if I was way off, and if this is even still bothering you, seeing that this post is from 2021
